15/umbraco-commerce/how-to-guides/member-portal.md

```diff
@@ -0,0 +1,235 @@
+2. Click on the **Members** tab in the navigation.
+3. Click on the Member you want to assign to the `Customer` Member Group.
+4. Select the `Customer` Member Group in the **Member Group** property.
+
+![Example Customer](images/member-portal/customer-assign-member-group.png)
+
+5. Click the **Save** button to assign the Member to the `Customer` Member Group.
+
+## Setting Up the Member Area
+
+The next step in building a members portal is to create the pages and templates that will make up the members area of your website.
+
+### Document Type Setup
+
+1. Navigate to the **Settings** section of the backoffice.
+2. Create two new Document Types: `Customer Portal` and `Login`.
+
+![Customer Portal Document Types](images/member-portal/document-types.png)
+
+3. Update your site root Document Type to include the `Customer Portal` and `Login` Document Types as child-pages.
+
+![Allowed Children Configuration](images/member-portal/allowed-children.png)
+
+### Content Setup
+
+1. Navigate to the **Content** section of the backoffice.
+2. Create a new page using the `Customer Portal` Document Type and name it `Customer Portal`.
+3. Create a new page using the `Login` Document Type and name it `Login`.
+
+![Customer Portal Content Structure](images/member-portal/content-structure.png)
+
+4. Expand the context menu for the `Customer Portal` node by clicking the three dots.
+5. Click on the **Public Access** option.
+6. Choose the **Group based protection** option in the **Public Access** dialog and select **Next**.
+7. Select the `Customer` Member Group for the group option.
+8. Select the `Login` node for the login and error page options.
+
+![Public Access Configuration](images/member-portal/public-access.png)
+
+9. Click **Save** to apply the public access settings.
+
+## Implementing a Member Login
+
+To access the members portal, customers need to log in. Through the following steps a login form allowing customers to enter their username and password to access the portal is created.
+
+1. Open the `Login.cshtml` template file.
+2. Add the following code to create a login form:
+
+```csharp
+@using (Html.BeginUmbracoForm<UmbLoginController>("HandleLogin", new { RedirectUrl = "/customer-portal" }))
+{
+    <div asp-validation-summary="ModelOnly"></div>
+        
+    <div>
+        <label asp-for="@loginModel.Username">Username</label>
+        <input asp-for="@loginModel.Username" />
+        <span asp-validation-for="@loginModel.Username"></span>
+    </div>
+    
+    <div>
+        <label asp-for="@loginModel.Password">Password</label>
+        <input asp-for="@loginModel.Password" />
+        <span asp-validation-for="@loginModel.Password"></span>
+    </div>
+    
+    <button type="submit">Login</button>
+}
+```
+
+{% hint style="info" %}
+The `UmbLoginController` class comes pre-installed with Umbraco and handles the login process for you, so you don't need to create a custom controller.
```

Comment:
```suggestion
The `UmbRegisterController` class comes pre-installed with Umbraco. It handles the login process, so you don't need to create a custom controller.
```

Reply:
Incorrect controller change, should be `UmbLoginController` but I'll update this along with the other suggestions.